### DISCUSSION TITLE
trainable lemmatizer with lookup tables

### DISCUSSION THREAD

**Discussion by jmyerston** [explosion/spaCy#11639] [New Features & Project Ideas]
hi, 

I wonder if you could allow the possibility of calling the lookup tables from the config section of the lemmatizer in a way similar to this:

```
[components.lemmatizer]
factory = "trainable_lemmatizer"
backoff = "lookup"
min_tree_freq = 6
overwrite = false
scorer = {"@scorers":"spacy.lemmatizer_scorer.v1"}
top_k = 1
```
As right now, calling the lookup tables is possible by creating both the trainable and lookup lemmatizers and setting backoff to null, but this seems verbose to me. 

Thanks.

**Comment by adrianeboyd** [comment]
We'd rather keep the lemmatizers more modular, since it's easier to combine different lemmatizers with different precedence.

With `backoff = null` in the trainable lemmatizer and `overwrite = false` in the following lookup lemmatizer, we feel like it's not too difficult to combine the two?
